frontend/src/app/login/login.component.css

```diff
@@ -1,8 +1,3 @@
-.card{
-    margin-top: 50px;
-    margin-bottom: 50px;
-}
```

Comment:
Can you leave this style in? I fixed the issue where it is left aligned in a separate PR.